<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomeDialog</class>
 <widget class="QDialog" name="welcomeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(22, 25, 37);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" alignment="Qt::AlignTop">
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-top:10px;
margin-bottom:10px;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Welcome&lt;/span&gt; to Covid Dashboard&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_openRecent">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-left:20px;</string>
     </property>
     <property name="text">
      <string>Open recent</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" rowspan="4">
    <widget class="QListView" name="openRecentList">
     <property name="styleSheet">
      <string notr="true">QListView{background: transparent;  border:none; }</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_samples">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-left:20px;</string>
     </property>
     <property name="text">
      <string>Samples</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QListView" name="samplesList">
     <property name="styleSheet">
      <string notr="true">QListView{
	background: transparent;
	 border:none ;
}</string>
     </property>
     <property name="itemAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_newProject">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-left:20px;</string>
     </property>
     <property name="text">
      <string>New Project</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="startupCheckbox">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open at Startup&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Open at startup</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="newhorizontal">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="newfilebtn">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open New File&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>New File</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
